apps/web/src/server/applications/representations/application-representations.view-model.js

```diff
@@ -0,0 +1,83 @@
+import { format } from 'date-fns';
+
+/**
+ *
+ * @param {object} arg
+ * @param {string=} arg.organisationName
+ * @param {string=} arg.firstName
+ * @param {string=} arg.lastName
+ * @returns {string}
+ */
+const getTitle = ({ organisationName, firstName, lastName }) =>
```

Comment:
maybe you can find some of these functions already implemented in the nunjucks-filters

Reply:
wrong PR this is in draft. 

This need the PR first - https://github.com/Planning-Inspectorate/back-office/pull/832